docs/examples/driver_examples/Qcodes example with Keysight B1500 Parameter Analyzer.ipynb

```diff
@@ -807,6 +807,194 @@
     "constants.ComplianceStatus.C"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Performing open/phase compensation"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Before performing these compensation/calibration measurements make sure that **probe needle is not connected to the device**. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Phase compensation"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Before performing phase compensation make sure that all the channel outputs are enabled else instrument throws an error. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 29,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "b1500.cmu1.enable_outputs()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Also the phase compensation mode must set to manual. These two are the requirements for running the phase compensation.  "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 30,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "b1500.cmu1.phase_compensation_mode(constants.ADJ.Mode.MANUAL)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now the phase compensation can be performed as following. There three possible outputs after the phase compensation \n",
+    " - Passed \n",
+    " - Failed \n",
+    " - Aborted. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 31,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<Response.PASSED: 0>"
+      ]
+     },
+     "execution_count": 31,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "b1500.cmu1.phase_compensation()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Open compensation"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To perform correction data measurement. The argument of this function requires frequency to be added in Hz. The correction data will be invalid after this command. Also one must set the AC voltage. "
```

Comment:
could you please rewrite it in a more fluid-readable fashion? :)

Reply:
The notebook is reworked. Hopefully, this time the flow should feel better.